pkg/volume/csi/csi_block.go

```diff
@@ -435,6 +444,57 @@ func (m *csiBlockMapper) TearDownDevice(globalMapPath, devicePath string) error
 			return err
 		}
 	}
+	if err = m.cleanupOrphanDeviceFiles(); err != nil {
+		return err
```

Comment:
Should we keep trying to delete directories, here? Or can we just log and return without error as it is done in `SetUpDevice`?